modules/core/src/main/java/org/apache/synapse/rest/API.java

```diff
@@ -313,6 +316,16 @@ void process(MessageContext synCtx) {
 			((Axis2MessageContext) synCtx).getAxis2MessageContext().
 							setProperty(NhttpConstants.REST_URL_POSTFIX,restURLPostfix);
 		}
+        if (synCtx.isResponse()){
```

Comment:
Shall we fix the formatting issue here?

Reply:
fixed in https://github.com/wso2/wso2-synapse/pull/1305